assets/css/components/_kw-pagination.scss

```diff
@@ -0,0 +1,4 @@
+.kw-pagination {
```

Comment:
Similar comment in `_button_detail.scss`, if this one not reused anywhere. We can apply this class in the component directly.

Reply:
In this case, it's all the utility classes. So you can ignore this comment. 👍